src/surface-components/Card/Card.module.css

```diff
@@ -0,0 +1,92 @@
+/* Card.module.css */
+
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap");
```

Comment:
**issue:** Bundler will not compile it currently - you can check locally but I think we can remove it, and the font will stay anyway